book/chapters/chapter2/data_and_basic_modeling.qmd

```diff
@@ -937,7 +932,7 @@ plt = plt + ggplot2::scale_fill_manual(values = c("grey30", "grey50"))
 print(plt)
 ```
 
-Whilst this model may appear to have good performance on the surface, in fact, it just ignores all 'bad' customers -- this can create very big problems in healthcare and other settings where there are data biases, or for insurance companies if `r index('false positives')` cost more than `r index('false negatives')` (see @sec-cost-sens for cost-sensitive classification).
+Whilst this model may appear to have good performance on the surface, in fact, it just ignores all 'bad' customers -- this can also create very big problems here, or in healthcare tasks and other settings where `r index('false positives')` cost more than `r index('false negatives')` (see @sec-cost-sens for cost-sensitive classification).
```

Comment:
```suggestion
Whilst this model may appear to have good performance on the surface, in fact, it just ignores all 'bad' customers -- this can create big problems in this finance example, as well as in healthcare tasks and other settings where `r index('false positives')` cost more than `r index('false negatives')` (see @sec-cost-sens for cost-sensitive classification).
```